src/utils/env.ts

```diff
@@ -41,7 +49,7 @@ function findPcEnvironment(location: URL, targetEnv: string, parseEnvironment: E
     } else {
         for (const environment of __GC_DEV_EXTRA_ENVS__) {
             const publicDomains = [environment.publicDomainName, ...(environment.publicDomainAliases || [])];
```

Comment:
nit: missed this before but the `|| {}` bit should be unnecessary since the publicDomainAliases type is string[].

Will be moot if we combine these two blocks into one. 

Also is a nit and nbd ;)

Reply:
Yes Justin, I added this one in the case that the developer doesn't add `publicDomainAliases` to his `CLIENT_APP_SDK_GC_EXTRA_ENVS` variable. I'll remove this one and document the field.